### PR TITLE
Updated README to use relative image paths

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,11 +93,11 @@ gym
 :outbox_tray: | All archives are stored and accessible in the Xcode Organizer
 :computer: | Supports both iOS and Mac applications
 
-![/assets/gymScreenshot.png](/assets/gymScreenshot.png)
```
Comment (asfalcone) [inline README.md]: Need to fix the second part. There is still a slash in the link!

**Comment by mfurtak** [comment]
I believe you can leave the `![]` part empty when defining these images to cut down on the repetition :+1:

**Comment by KrauseFx** [comment]
The `[text]` is the `alt` text when the image doesn't get loaded and for screen readers.

**Comment by KrauseFx** [comment]
@asfalcone updated :+1:

**Comment by asfalcone** [comment]
LGTM! :hatched_chick:

**Pull request merged**